.github/workflows/build-nym-vpn-desktop-linux.yml

```diff
@@ -42,7 +42,12 @@ jobs:
 
       - name: Install system dependencies
         run: |
-          sudo apt-get update && sudo apt-get install -y libdbus-1-dev libmnl-dev libnftnl-dev protobuf-compiler libwebkit2gtk-4.0-dev build-essential curl wget libssl-dev libgtk-3-dev squashfs-tools libayatana-appindicator3-dev make libfuse2 librsvg2-dev file
+          sudo apt-get update && sudo apt-get install -y libdbus-1-dev libmnl-dev libnftnl-dev libwebkit2gtk-4.0-dev build-essential curl wget libssl-dev libgtk-3-dev squashfs-tools libayatana-appindicator3-dev make libfuse2 librsvg2-dev file
+
+      - name: Install protoc
```

Comment:
So I'm not convinced about this, the implications are that we can't anymore build on plain Ubuntu install. What are the reasons for upgrading?

Reply:
We need to upgrade to support grpc optionals for credential import. The deb package is outdated. Additionally, I think it is better and easier to maintain if we are using the same install/setup and version on all of our workflows.

Reply:
It's logic to rely on the latest stable protoc version…

Not logic to rely on whatever Ubuntu is using as protoc version from its system package (which is outdated on runners)…

…to build our inter process protocol communication!

Reply:
If whatever package system you use is outdated, it is very straight to switch on latest stable simply by downloading it from official releases -> https://github.com/protocolbuffers/protobuf/releases/tag/v27.2

(a matter of putting `protoc`  in the PATH)

Reply:
Discussed extensively elsewhere, but adding here for completeness. I would prefer if we keep system `protobuf-compiler` on ubuntu.

However, a possible way forward here is if add `ubuntu-24.04` to `ci-nym-vpn-core.yml` with system protoc, and switch to manually installed protoc on ubuntu-22.04. Then we can start using `optional` while still keeping keeping the integrity of the system installed toolchain.